### PR TITLE
ISSUE-4242: Use `database_id/table_id` as data file prefix

### LINKED ISSUE #4242: Add `database_id/table_id` to data file prefix

### PR DESCRIPTION
**Pull request by dantengsky**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

implements 
- issue #4242 

## Changelog

- Improvement
- Not for changelog (changelog entry is not required)

## Related Issues

Fixes #4242

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

File: query/src/interpreters/interpreter_table_create.rs [hunk 1]
```diff
@@ -60,28 +61,56 @@ impl Interpreter for CreateTableInterpreter {
...
                     .await
             }
-            None => self.create_table().await,
+            None => self.create_table(create_plan).await,
         }
     }
 }
 
 impl CreateTableInterpreter {
+    async fn plan_with_db_id(&self) -> Result<CreateTablePlan> {
```
Comment (BohuTANG) [inline query/src/interpreters/interpreter_table_create.rs:77]: Add the db_id to the Plan in statements/statement_create_table.rs?

Comment (dantengsky) [reply query/src/interpreters/interpreter_table_create.rs:77]: :ok_hand:

Comment (BohuTANG) [reply query/src/interpreters/interpreter_table_create.rs:77]: Only this little comment, Others LGTM , thank you man!

Comment (dantengsky) [reply query/src/interpreters/interpreter_table_create.rs:77]: got it. still working on it, will be pushed soon.

Comment (dantengsky) [reply query/src/interpreters/interpreter_table_create.rs]: @BohuTANG  PTAL

**Pull request merged**